src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  function adder() {
+    if (Object.values(arguments).length === 0) {
```

Comment:
probably more comfortable without objectValues)))

Reply:
the taste and color of all felt-tip pens are different